app/src/main/res/values/strings.xml

```diff
@@ -764,6 +764,10 @@
     <string name="history_older">Older</string>
     <!-- Text shown when no history exists -->
     <string name="history_empty_message">No history here</string>
+    <!-- Option in library for opening Synced History screen. History that is coming from other devices -->
+    <string name="library_synced_history">Synced from other devices</string>
+    <!--  The page title for browsing history coming from other devices. -->
+    <string name="synced_history">From other devices</string>
```

Comment:
```suggestion
    <string name="history_from_other_devices">From other devices</string>
```

Reply:
Followed the pattern from the element next to it – Recently closed tab:
```
<!-- Option in library for Recently Closed Tabs -->
    <string name="library_recently_closed_tabs">Recently closed tabs</string>
```

Reply:
Can you move it besides `library_recently_closed_tabs` then? We want this to be sorted by the prefix ideally or section ideally.

Reply:
It sits inside the history section already.

I don't know about the concept and it is a bit confusing to me. Will keep the place, but change the name to the suggested one